analyzers/src/SonarAnalyzer.VisualBasic/Rules/MultipleVariableDeclarationCodeFix.cs

```diff
@@ -18,79 +18,50 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CodeFixes;
 using Microsoft.CodeAnalysis.Formatting;
-using Microsoft.CodeAnalysis.VisualBasic;
-using Microsoft.CodeAnalysis.VisualBasic.Syntax;
 
 namespace SonarAnalyzer.Rules.VisualBasic
 {
     [ExportCodeFixProvider(LanguageNames.VisualBasic)]
     public sealed class MultipleVariableDeclarationCodeFix : MultipleVariableDeclarationCodeFixBase
     {
-        protected override SyntaxNode CalculateNewRoot(SyntaxNode root, SyntaxNode node)
-        {
-            if (!(node is ModifiedIdentifierSyntax identifier))
-            {
-                return root;
-            }
+        protected override SyntaxNode CalculateNewRoot(SyntaxNode root, SyntaxNode node) =>
+            node is ModifiedIdentifierSyntax { Parent: VariableDeclaratorSyntax declarator }
+                ? root.ReplaceNode(declarator.Parent, CreateNewNodes(declarator))
+                : root;
```

Comment:
Positive logic... this is soo much better

Reply:
Yep, in general, I'm a person with positive thinking :P